CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
     Click to see more.
   </summary>
 
+### Major
+
+- Box: BREAKING CHANGE - Removed support for `shape` prop values `roundedTop`, `roundedRight`, `roundedBottom`, and `roundedLeft` (#657)
```

Comment:
@jennyscript could we use less caps? Perhaps use `[Breaking]`? Also, is it truly breaking if we provide a codemod?

Reply:
I can add a codemod, but as we discussed it won't have exactly the same behavior as before (either rounded or square subsistitute) so I'd still consider that breaking someone's code.

Reply:
@jennyscript Yes, it would be ideal to still add the codemod and do a major bump. The codemod will avoid JavaScript / flow and proptype errors.